src/App.tsx

```diff
@@ -1,52 +1,70 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import {
+  Navigate, Route, Routes,
+} from 'react-router-dom';
+import { useState } from 'react';
+import { MainNav } from './Components/MainNav';
+import { TabsPage } from './Components/TabsPage';
+import { Tab } from './types/Tab';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
-export const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
-      data-cy="Nav"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
-        </div>
-      </div>
-    </nav>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState<Tab | undefined>(undefined);
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState<Tab | undefined>(null);
```
in such cases better to use `null`